docs/deployment.md

```diff
@@ -0,0 +1,214 @@
+# Deployment of Forcerelay/Axon
+
+dependencies and tools:
+1. Axon Rpc (Alphanet): https://rpc-alphanet-axon.ckbapp.dev
+2. CKB Rpc (Testnet): https://testnet.ckbapp.dev/
+3. Yarn: v1.22.19
+4. ckb-cli: v1.4.0
+5. cargo: v1.74,0
+
+related repos and commits:
+1. ibc-solidity-contract: https://github.com/synapseweb3/ibc-solidity-contract
+2. forcerelay-ckb-contracts: https://github.com/synapseweb3/forcerelay-ckb-contracts
+3. forcerelay-ckb-sdk: https://github.com/synapseweb3/forcerelay-ckb-sdk
+
+## Deploy `ibc-solidity-contracts`
+deploy from the repo (Yarn needed):
+```bash
+$ git clone https://github.com/synapseweb3/ibc-solidity-contract
+$ cd ibc-solidity-contract
+$ yarn install
+$ yarn compile
+$ AXON_HTTP_RPC_URL=https://rpc-alphanet-axon.ckbapp.dev yarn migrate
+```
+ 
+after the deployment, contracts `OwnableIBCHandler` and `ICS20TransferERC20` are in your output, let's record the addresses of them as **<IBC_HANDLER_ADDRESS>** and **<TRANSFER_ERC20_ADDRESS>** for a later use.
+
+## Deploy `forcerelay-ckb-contracts`
+
+we have compiled all of the latest 4 ckb contracts, named `Connection`, `Channel`, `Packet` and `sudt-transfer`, in the repo of Forcerelay, please make sure there's enough CKB capacity in your ckb address **<your_ckb_address>** and follow the below steps (ckb-cli needed):
+```bash
+$ git clone https://github.com/synapseweb3/forcerelay
+$ cd forcerelay/tools/ibc-test/contracts/deployment
+$ ./upload.sh connection https://testnet.ckbapp.dev/ <your_ckb_address>
+$ ./upload.sh channel https://testnet.ckbapp.dev/ <your_ckb_address>
+$ ./upload.sh packet https://testnet.ckbapp.dev/ <your_ckb_address>
+$ ./upload.sh sudt https://testnet.ckbapp.dev/ <your_ckb_address>
+```
```

Comment:
Does metadata also need to be deployed?

Reply:
it is necessary to supplement and update the lock.args corresponding to the deployment contract account.

Reply:
> Does metadata also need to be deployed?

In real case, metadata cell is not deployed that way like these IBC contracts do